README.md

```diff
@@ -96,6 +96,18 @@ destroy` or turn off all services with `lando poweroff`.
      - Or run services separately as shown in [[https://github.com/pulibrary/figgy/blob/master/Procfile]]
    - Access Figgy at http://localhost:3000/
 
+### Parallel Tests
+
+If you'd like to run the test suite in parallel do the following:
+
+1. `bundle exec rake servers:start`
+1. `PARALLEL_TEST_FIRST_IS_1=true RAILS_ENV=test rake parallel:setup`
```

Comment:
This is only needed once? Should it be integrated into `.bin/setup`?

It's not clear from the layout whether this is a prerequisite for both `./bin/turbo_test` and `parallel_rspec`. The `parallel_rspec` list item starts over at 1 again due to the text between the two list items. Is that intended? Should the third list item have 2 sub-items with the 2 different ways to run the tests?